src/monitor/utils/monitor.factory.ts

```diff
@@ -0,0 +1,59 @@
+import { Monitor } from '../domain/monitor';
+
+export class MonitorFactory {
+  static createMonitor(prismaMonitor): Monitor {
+    const student = {
+      id: prismaMonitor.student.user.id,
+      name: prismaMonitor.student.user.name,
+      contactEmail: prismaMonitor.student.contact_email,
+      whatsapp: prismaMonitor.student.whatsapp,
+      linkedin: prismaMonitor.student.linkedin,
+    };
+
+    const subject = {
+      id: prismaMonitor.subject.id,
+      code: prismaMonitor.subject.code,
+      name: prismaMonitor.subject.name,
+      course: {
+        id: prismaMonitor.subject.course.id,
+        code: prismaMonitor.subject.course.code,
+        name: prismaMonitor.subject.course.name,
+      },
+    };
+
+    const responsible = {
+      id: prismaMonitor.responsible_professor.user.id,
+      name: prismaMonitor.responsible_professor.user.name,
+      email: prismaMonitor.responsible_professor.user.email,
+    };
+
+    const monitorSettings = {
+      id: prismaMonitor.MonitorSettings.id,
+      preferentialPlace: prismaMonitor.MonitorSettings[0].preferential_place,
+      isActive: prismaMonitor.MonitorSettings[0].is_active,
+    };
```

Comment:
Você precisa tratar isso como opcional:

```suggestion
    let monitorSettings = null;
    if (prismaMonitor.MonitorSettings.length) {
      monitorSettings = {
        id: prismaMonitor.MonitorSettings[0].id,
        preferentialPlace: prismaMonitor.MonitorSettings[0].preferential_place,
        isActive: prismaMonitor.MonitorSettings[0].is_active,
      };
    }
```